api/src/prescription/campaign/domain/read-models/OrganizationMembership.js

```diff
@@ -0,0 +1,7 @@
+class OrganizationMembership {
+  constructor({ isAdmin } = {}) {
+    this.isAdmin = isAdmin;
+  }
+}
+
+export { OrganizationMembership };
```

Comment:
**question** est-il util que l'on se crée un objet du domain ? Sachant que organizational entities expose un objet du domain spécifique pour son API ?

Reply:
Repondu à l'oral : Concretement, on ne veut pas avoir à utiliser directement les objets d'un autre context. Celui la est "simple" mais on pourrait avoir besoin de rajouter des regles ou autre qui du coup appartiendrait à celui de notre context. De plus si du coté organization entities l'objet venait à évoluer, il ne dois pas se refléter dans notre implémentation. Ça nous ajoute une couche de "protection". 

**La notion d’ACL dans le read book :**
- *Anticorruption Layer: Translation layers can be simple, even elegant, when bridging well-designed Bounded Contexts with cooperative teams. But when control or communication is not adequate to pull off a shared kernel, partner, or customer-supplier relationship, translation becomes more complex. The translation layer takes on a more defensive tone. As a downstream client, create an isolating layer to provide your system with functionality of the upstream system in terms of your own domain model. This layer talks to the other system through its existing interface, requir- ing little or no modification to the other system. Internally, the layer translates in one or both directions as necessary between the two models.*